format/Schema.fbs

```diff
@@ -183,10 +183,27 @@ table Timestamp {
 }
 
 enum IntervalUnit: short { YEAR_MONTH, DAY_TIME}
+// A "calendar" interval which models types that don't necessarily
+// have a precise duration without the context of a base timestamp (e.g.
+// days can differ in length during day light savings time transitions).
+// YEAR_MONTH - Indicates the number of elapsed whole months, stored as 
+//   4-byte integers. 
+// DAY_TIME - Indicates the number of elapsed days and milliseconds,
+//   stored as 2 contiguous 32-bit integers (8-bytes in total).  Support
+//   of this IntervalUnit is not required for full arrow compatibility.
 table Interval {
   unit: IntervalUnit;
 }
 
+/// Indicates absolute time offset according to the difference
+/// in UNIX time (i.e. excluding leap seconds). The resolution defaults to
+/// millisecond, but can be any of the other supported TimeUnit values as
+/// with Timestamp and Time types.  This type is always represented as
+/// an 8-byte integer.
+table DurationInterval {
```

Comment:
Looking over the old e-mail threads it might be that TimeDelta or TimestampDelta is a better name for this type?

Reply:
Are you happy with "DurationInterval"? It could also be called simply "Duration" or "TimeDuration"